frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_list_item.tsx

```diff
@@ -417,130 +424,158 @@ function _SegmentListItem({
     return centeredSegmentId === segment.id;
   });
 
-  const createSegmentContextMenu = (): MenuProps => ({
-    items: [
-      getLoadPrecomputedMeshMenuItem(
-        segment,
-        currentMeshFile,
-        loadPrecomputedMesh,
-        hideContextMenu,
-        visibleSegmentationLayer != null ? visibleSegmentationLayer.name : null,
-        mappingInfo,
-        activeVolumeTracing,
-      ),
-      getComputeMeshAdHocMenuItem(
-        segment,
-        loadAdHocMesh,
-        visibleSegmentationLayer != null,
-        hideContextMenu,
-      ),
-      getMakeSegmentActiveMenuItem(
-        segment,
-        setActiveCell,
-        activeCellId,
-        isEditingDisabled,
-        hideContextMenu,
-      ),
-      {
-        key: `changeSegmentColor-${segment.id}`,
-        label: (
-          <ChangeColorMenuItemContent
-            isDisabled={false}
-            title="Change Segment Color"
-            onSetColor={(color, createsNewUndoState) => {
-              if (visibleSegmentationLayer == null) {
-                return;
-              }
-              updateSegment(
-                segment.id,
-                {
-                  color,
-                },
-                visibleSegmentationLayer.name,
-                createsNewUndoState,
-              );
-            }}
-            rgb={Utils.take3(segmentColorRGBA)}
-          />
+  const createSegmentContextMenu = (): MenuProps => {
+    const segmentColorWithMeshOpacity: Vector4 = [
+      segmentColorRGBA[0],
+      segmentColorRGBA[1],
+      segmentColorRGBA[2],
+      mesh != null ? mesh.opacity : Constants.DEFAULT_MESH_OPACITY,
+    ];
+    return {
+      items: [
+        getLoadPrecomputedMeshMenuItem(
+          segment,
+          currentMeshFile,
+          loadPrecomputedMesh,
+          hideContextMenu,
+          visibleSegmentationLayer != null ? visibleSegmentationLayer.name : null,
+          mappingInfo,
+          activeVolumeTracing,
         ),
-      },
-      {
-        key: "resetSegmentColor",
-        disabled: segment.color == null,
-        onClick: () => {
-          if (visibleSegmentationLayer == null) {
-            return;
-          }
-          updateSegment(
-            segment.id,
-            {
-              color: null,
-            },
-            visibleSegmentationLayer.name,
-            true,
-          );
+        getComputeMeshAdHocMenuItem(
+          segment,
+          loadAdHocMesh,
+          visibleSegmentationLayer != null,
+          hideContextMenu,
+        ),
+        getMakeSegmentActiveMenuItem(
+          segment,
+          setActiveCell,
+          activeCellId,
+          isEditingDisabled,
+          hideContextMenu,
+        ),
+        {
+          key: `changeSegmentColor-${segment.id}`,
+          label: mesh?.isVisible ? (
+            <ChangeRGBAColorMenuItemContent
+              isDisabled={false}
```

Comment:
If isDisabled is always false for the `ChangeRGBAColorMenuItemContent` I'd say this prop can be removed imo

Reply:
right!!!